src/main/webapp/js/adminAccountManagement.js

```diff
@@ -1,6 +1,6 @@
 var entryPerPage = 200;
 
-var start = 0;
+var begin = 0;
```

Comment:
I don't know why but the variable name `start` causes the rule `no-native-reassign` to be violated, so I need to make the change here.